tests/typ/text/lang-with-region.typ

```diff
@@ -0,0 +1,6 @@
+#set text(lang: "zh")
```

Comment:
test files should generally start with a short comment and then `---` before the first test (you can see it in almost all other tests)